docs/building.md

```diff
@@ -48,19 +48,19 @@ make out/trivialDAO_storage.tz \
   guardian_address=KT1QbdJ7M7uAQZwLpvzerUyk7LYkJWDL7eDh \
   governance_token_address=KT1RdwP8XJPjFyGoUsXFQnQo1yNm6gUqVdp5 \
   governance_token_id=0n \
-  now_val=`date +"%s"` \
+  current_level=100n \
   metadata_map=(Big_map.empty : metadata_map) \
   fixed_proposal_fee_in_token=0n \
   ledger=([] : ledger_list) \
   quorum_threshold=10n \
   min_quorum=1n \
   max_quorum=99n \
-  period=950400n \
+  period={ blocks = 15840n } \
```

Comment:
It looks like this is incorrect, the `Makefile` takes the `nat` as an argument (as it should, IMO)

Reply:
Fixed.